net/common.sh

```diff
@@ -12,7 +12,7 @@ netDir=$(
   echo "$PWD"
 )
 netConfigDir="$netDir"/config
-netLogDir="$netDir"/log
+netLogDir="$netDir"/log/$(date +"%Y-%m-%d_%H_%M_%S")
```

Comment:
How about log-$(date ...)/ and then log/ becomes a symlink to the latest log directory?

Reply:
yea, good idea

Reply:
@mvines @danpaul000 I added that, take another look?